index.js

```diff
@@ -43,6 +43,17 @@ app.delete('/delete/:id', (req, res) => {
 });
 
 
+app.post('/create', (req, res) => { // take note of front end results if using new id such as 101 in url
+  db.addAuthorAndBook((err) => {
```

Comment:
Following best coding practice and separation of concerns, the data to be Added should be passed to the db.addAuthorAndBooks. That data could have come from the client (through postman) or generated on the server (with createfakeauthors & createfakebooks method)